catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -68,10 +77,31 @@
 
     private boolean enableDelete = true;
 
+    private boolean initialized = false;
+
+    private ScheduledExecutorService executor;
+
+    private int taskWaitTimeSeconds = 30;
```

Comment:
Since`taskWaitTimeSeconds`, `refreshInterval` (and `initialized` and/or `initialized`) are accessed by multiple threads, should we make them `volatile`?